mavros_extras/src/plugins/odom.cpp

```diff
@@ -49,14 +51,15 @@ class OdometryPlugin : public plugin::PluginBase {
 	}
 
 private:
-	ros::NodeHandle _nh;
+	ros::NodeHandle odom_nh;
 	ros::Subscriber _odom_sub;
```

Comment:
`nh` is fine. I don't like starting `_`, prefer `m_` instead. But that isn't a problem.